frontend/src/views/v-authorship.vue

```diff
@@ -579,6 +595,11 @@ export default {
       return window.getBlameLink(this.info.repo, repo.branch, file.path);
     },
 
+    isBrokenLink(link) {
```

Comment:
@LuoZhijie-tom This function appeared 3 times across 3 different files. I think it will be better to extract them out into a util file and import it. This way it is much easier to maintain and trace in the future if we want to change this function. I will suggest adding the string `This remote link is unsupported` to this new file as well.

You can probably add  a new file under `src/utils`.

Reply:
Hi @Zhou-Jiahao-1998 I have put this function into a file under src/utils, could you have a look?